ux.symfony.com/assets/styles/_images.scss

```diff
@@ -10,42 +10,8 @@
     border-radius: 12px;
 }
 
-.nav-component-img {
-    width: 20px;
-    height: 20px;
-    border-radius: 3px;
+.img-dark-negative {
+    [data-bs-theme="dark"] & {
+        filter: invert(100%) hue-rotate(180deg);
+    }
 }
-
-.testimonial-img {
-    border-radius: 50px;
-}
-
-.logo {
-    background: url(../images/sf-ux-logo.svg);
-    height: 30px;
-    width: 161px;
-}
-
-.logo-white {
-    background: url(../images/sf-ux-logo-white.svg);
```

Comment:
Looks like this file isn't used anymore, right? But I think it wasn't deleted yet.

Reply:
My mistake, fixed